backend/json_schema/hanko.config.json

```diff
@@ -675,14 +675,19 @@
         },
         "secret": {
           "type": "string"
+        },
+        "allow_linking": {
+          "type": "boolean",
+          "default": true
         }
       },
       "additionalProperties": false,
       "type": "object",
       "required": [
         "enabled",
         "client_id",
-        "secret"
+        "secret",
+        "allow_linking"
```

Comment:
I'am confused, the default value of `allow_linking` (currently) is set to `true` for all providers, why is this a required field?